manifests/params.pp

```diff
@@ -109,7 +109,11 @@
       $psql_path           = pick($psql_path, "${bindir}/psql")
 
       $perl_package_name   = pick($perl_package_name, 'perl-DBD-Pg')
-      $python_package_name = pick($python_package_name, 'python-psycopg2')
+      if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '8') >= 0 {
```

Comment:
Just thinking out loud: this will also cover Fedora. It will break Fedora versions which are still primarily Python 2 but those are EOL anyway. Probably not a big deal.